docs/changelog.rst

```diff
@@ -6,6 +6,11 @@ Changelog
 `Unreleased`_
 -------------
 
+Added
+~~~~~
+
+- Support PEP561 `#748`_
```

Comment:
Could you please add `.. _#748: https://github.com/schemathesis/schemathesis/issues/748` string to the end of this file where other links are placed? It will make the rendered changelog point to the proper issue location.